### PR TITLE
Door security increase

### PR DESCRIPTION
**Pull request by Flakkyboo**
## About The Pull Request
Space OSHA have done a review of centcoms wiring practices and found them sorely lacking and requiring an update.

Airlocks for command, security, engineering, science, medical, cargo will now have randomized wiring that will only be common to that department.

Increased the security of command level doors now requiring a weld before gaining access to the wires.

Port of https://github.com/Citadel-Station-13/Citadel-Station-13/pull/5574

## Why It's Good For The Game
No more speedrunning captains spare in 2 minutes because you learned the test wire from a public door.

Also makes hacking less of a skeleton key and more of an involved practice in remembering each test wire 

## Changelog
:cl:
add: Adds code allowing for the randomizing the wires of individual door types
balance: Increased command door security level from 0 to 1
/:cl:

### CODE REVIEW

**Comment by Bestparty** [comment]
never

File: code/game/machinery/doors/airlock_types.dm [hunk 1]
```diff
@@ -5,20 +5,25 @@
 /obj/machinery/door/airlock/command
 	icon = 'icons/obj/doors/airlocks/station/command.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_com
+	wiretypepath = /datum/wires/airlock/command
 	normal_integrity = 450
+	security_level = 3
```
Comment (zeskorion) [inline code/game/machinery/doors/airlock_types.dm]: is this plasteel(cutter, weld, pry, weld, pry), or just iron?

Comment (zeskorion) [reply code/game/machinery/doors/airlock_types.dm]: If it's plasteel, that's literally vault door level. Reduce it to iron

Comment (Flakkyboo) [reply code/game/machinery/doors/airlock_types.dm]: plasteel (screw,weld,pry)
level 2 security is plasteel (screw,pry)
level 1 security is iron (screw, weld)

Comment (zeskorion) [reply code/game/machinery/doors/airlock_types.dm]: Like i said, reduce it to iron

**Comment by St0rmC4st3r** [comment]
Are the different wire schemes listed on the station blueprints?

**Comment by Flakkyboo** [comment]
> 
> 
> Are the different wire schemes listed on the station blueprints?

They are yep

File: code/game/machinery/doors/airlock_types.dm (hunk 1 above)
Comment (Flakkyboo) [reply code/game/machinery/doors/airlock_types.dm]: i didnt see the comment update im sorry.
sure ill push that

**Review by St0rmC4st3r — APPROVED** [review]

**Comment by St0rmC4st3r** [comment]
Travis broke, everything else looks good to me.
Or at least I am sure that `runtime error: Global var internal_tick_usage does not exist.` is not your fault.

**Comment by Flakkyboo** [comment]
> 
> 
> Travis broke, everything else looks good to me.
> Or at least I am sure that `runtime error: Global var internal_tick_usage does not exist.` is not your fault.

yeah all i changed from the build that passed to this one was change a 3 to a 1

**Comment by TeomanTheGreat** [comment]
No Flakky don't do it

**Comment by Bastian0930** [comment]
haha travis go brrr

**Comment by Flakkyboo** [comment]
Travis is an eternal thorn in my side

**Review by zeskorion — APPROVED** [review]

**Comment by ACanOpener** [comment]
This is bad and I hate this and you should feel bad

**Comment by Mick1299** [comment]
How do freshly constructed doors pick their wiretype? Is it based on the access or colour, or does it just default to the universal default wiretype?

**Comment by St0rmC4st3r** [comment]
The smell you feel? It is shame. Another good PR downvoted for convenience of tiding.

**Comment by Flakkyboo** [comment]
Truly it is sad.
And I think it shows nothing that makes tiding harder will pass the mob mentally

**Pull request closed without merging**